### ISSUE TITLE
Pull requests not being reviewed.

### ISSUE THREAD

**Issue by stevepeak** [storyscript/community#12]
tl:dr; Some pull requests were not reviewed showing a failure in process around reviewing and accepting contributions.

### Notes

@jayvdb identified stale pulls that were not reviewed or merged. These three are the smallest examples and very simply to resolve, but ignored from our team.

- 6 day, no review, https://github.com/storyscript/docs.storyscript.io/pull/135
- 6 day, no review, https://github.com/storyscript/docs.storyscript.io/pull/136
- over a month ago, no merge, https://github.com/storyscript/docs.storyscript.io/pull/125

This laps of pull review/merge is not healthy for our community development. Pulls should be addressed like a user-facing issue, taking first priority. 

### Result
We have many repositories and rely on people being owners of project is not scalable, replaceable and has to much opportunity for fault.

We need to consider tools, policies and best practices around resolving this issue.

----

**References**
- List all related issues in GitHub or external links here.

> This meeting took place in office with the following attendees: @StoryScriptAI @judepereira @JeanBarriere @stevepeak @williammartin @rohit121 @Arinono

**Comment by wilzbach** [comment]
> over a month ago, no merge, storyscript/docs.storyscript.io#125

FWIW the accompanying CLI PR hasn't been merged for this PR at that time, so it couldn't be merged into the live docs.
See: https://github.com/storyscript/cli/pull/240

**Comment by stevepeak** [comment]
@wilzbach thank you for clarifying, for this particular case we should have referenced the issue and set the pull request to a pending state so it's very clear to contributors.

**Comment by stevepeak** [comment]
@StoryScriptAI what were the bots you suggested? This could also be a really nice mini-app to build in Storyscript: monitoring our development policies and reporting details to our dashboard.

**Comment by jayvdb** [comment]
A good way to help manage many repositories is to have org level labels and processes around them.

e.g. the coala review queue is https://coala.io/review , https://coala.io/approved , etc . It used to hum well, but the maintainers have been busy with jobs, and we used to have a bot automatically doing the labeling , but that was based on @gitmateio which is now a discontinued service .  The coala labels are at https://github.com/coala/coala/labels (coala did npt use org level, but we had a 'new repo' script which created them)

An equivalent should be easy to set up.

If there is interest in this, I could create a separate issue here about labelling, where we choose a top level scheme that suits the team, and also find bots which can assist.

**Comment by jayvdb** [comment]
In addition to lack of reviews, the larger problem is opinions from reviewers that are ill-informed and/or written in a way that does not invite discussion around the area of contention in order to find the best solution.

https://github.com/storyscript/hub-sdk-python/pull/32 is a classic example of this.  I opened a thread about Python "3.5" as I found inconsistencies everywhere - that aspect was ignored.  But then the PR was approved, but with disagreement on core elements of the patch.  What message does that send to the coder?  Confusion.

Also there is need for a policy of who can merge, and when, how, etc.  Maybe there is a policy already, or unwritten expectations in place, but that needs to be formalised a bit to help newcomers find their feed.  Also prior development practises need to be revisited over the coming months to ensure they are still appropriate as the product should be moving from rapid development to stable and secure and maintainable, and user-friendly and newcomer-developer-friendly.  

Not merging your own work is a typical element of such a policy, intended to ensure there is at least someone else who agrees with the changes.  However some allowance might be made for trivial changes (defining this is hard), and also 'single developer projects' where constraints will be counter-productive as no community has yet formed around the project.
